pkg/provider/cloud/openstack/provider.go

```diff
@@ -223,10 +236,23 @@ func (os *Provider) InitializeCloudProvider(cluster *kubermaticv1.Cluster, updat
 	}
 
 	if cluster.Spec.Cloud.Openstack.SecurityGroups == "" {
-		secGroupName, err := createKubermaticSecurityGroup(netClient, cluster.Name)
+		lowPort, highPort := resources.NewTemplateDataBuilder().
+			WithNodePortRange(osp.nodePortsRange).
+			WithCluster(cluster).
+			Build().
+			NodePorts()
```

Comment:
Why we don't use the `ComputedNodePortRange` function here? My understanding is that the `NodePorts` function doesn't take component overrides into account, or if the provided range is an empty string.

Reply:
NodePorts() uses ComputedNodePortRange() to get string to parse (and return ints for low and high ports), so it does take override logic into account.